src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -74,6 +75,16 @@ public void execute_multipleKeywords_multiplePersonsFound() {
         assertEquals(Arrays.asList(CARL, ELLE, FIONA), model.getFilteredPersonList());
     }
 
+    @Test
```

Comment:
Need to add Tests for NameContainsKeywordsPredicateTest and FindCommandParserTest, the same tests as above, to ensure complete coverage of the affected code